### ISSUE TITLE
[TW-407] please silence "TASKRC override: ..." messages

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#442]
_Jakub Wilk on 2013-02-26T18:04:14Z says:_ 

 I don't like my $HOME polluted with dotfiles, so I set the TASKRC environment variable. But now every time I run task, it print this message:

TASKRC override: /path/to/my/taskrc

Well, I set this variable myself. I don't need to reminded about it every time...
Please silence these messages (perhaps as an option). Thanks for considering.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-02-26T18:04:14Z
Modified: 2014-02-09T01:44:44Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-03-02T23:29:17Z says:_ 

 Already supported.

The message can be silenced by removing 'header' as a verbosity token.  See 'man taskrc',  look for 'verbose='.

**Comment by taskwarrior** [comment]
_Jakub Wilk on 2013-03-08T18:17:00Z says:_ 

 Is "TASKRC override: ..." the only message that is classified as "header"?
I doubt it is, but I couldn't find any documentation on this subject.

**Comment by jcrben** [comment]
I have a similar concern - this came up over at https://unix.stackexchange.com/questions/367151/how-to-override-warning-in-taskwarrior and similar concerns were raised about this possibly silencing other messages

**Comment by ulope** [comment]
Silencing a whole class of messages to suppress one mostly useless message seems more like a hack rather than fix...

**Comment by pbeckingham** [comment]
Suggestions?  PR?

**Comment by jdorel** [comment]
Fixed by #2077
